login.php

```diff
@@ -1,6 +1,6 @@
 <?php
 session_start();
-if ($_SESSION["email"] != "") {
+if (isset($_SESSION["email"])) {
```

Comment:
`isset` and being an empty string are different things. You can check for both, as in the logout file, you have probably set the session variable to an empty string, and not `unset` it.